src/zcl_aff_writer_xslt.clas.abap

```diff
@@ -304,17 +314,23 @@ CLASS zcl_aff_writer_xslt IMPLEMENTATION.
     IF abap_doc-callback_class IS NOT INITIAL AND is_callback_class_valid( class_name = abap_doc-callback_class component_name = fullname_of_type ).
       write_callback_template( element_name = element_name description = element_description tag = tag ).
     ENDIF.
-
-
     write_open_tag( |<tt:cond{ get_condition_for_element( element_name = element_name element_description = element_description enum_values = enum_values type = type ) }>| ).
     write_open_tag( |<{ tag }{ get_name( name = element_name ) }>| ).
     IF ( is_sy_langu( element_description = element_description ) ).
       write_iso_language_callback( element_name = element_name ).
     ELSEIF enum_values IS INITIAL.
       write_tag( |<tt:value{ get_ref( element_name ) }{ get_option( json_type = type element_description = element_description ) }/>| ).
     ELSE.
-      write_enum_value_mappings( element_description = element_description json_type = type element_name = element_name enum_values = enum_values ).
+      write_enum_map_ext_compatible(
+        element_description = element_description
+        element_name        = element_name
+        enum_values         = enum_values ).
+      write_open_tag( |<tt:serialize>| ).
```

Comment:
The deserialize-tag is written in the write_enum_map_ext_compatible, the serializa-tag outside. Should we keep this in sync? Maybe both outside? What do you think?

Reply:
yeah, i think outside is better